ports/stm32/Makefile

```diff
@@ -64,6 +64,7 @@ OPENOCD_CONFIG ?= boards/openocd_stm32f4.cfg
 include stm32.mk
 
 PLLVALUES = boards/pllvalues.py
+I2SPLLTABLE = boards/i2splltable.py
```

Comment:
I suggest naming the new script `pllvalues_i2s.py`, to match the existing one.

Reply:
The I2S PLL is referred to as `plli2s` throughout the stm32 reference docs.  This led to the choice of `plli2svalues.py`.  This naming choice seems reasonably consistent with `pllvalues.py`.